examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -72,7 +128,7 @@
    "cell_type": "markdown",
```

Comment:
It would be useful to display the result, ideally as a graph in which the solution nodes are outlined.
Both here and below.

(this could be implemented as a method of `MISSolution`)

Reply:
UnitaryHack ?

Reply:
Sounds good, yes.